Test/DafnyTests/DafnyTests.csproj

```diff
@@ -43,6 +43,8 @@
     <PackageReference Include="dafny.msbuild" Version="0.1.2" />
     <PackageReference Include="xunit" Version="2.4.1" />
     <PackageReference Include="xunit.runner.msbuild" Version="2.4.1" />
+    <PackageReference Include="System.Collections.Immutable" Version="1.7.0" />
+    <PackageReference Include="System.Runtime" Version="4.3.1" />
```

Comment:
I think this is good enough for now, but I'd love to make these dependencies of `dafny.msbuild` instead! It's possible we'll shift to the DafnyRuntime package soon enough which will have those dependencies instead though.

Reply:
I can explore this a bit locally, but I wanted to timebox this, since we've already been spending several days on getting imports working, and it sounds like, going forward, we'll potentially be shifting how these imports are handled anyways.

Reply:
Yup, definitely not blocking, just an FYI. But be aware it means all dafny.msbuild clients will need to add these dependencies. It means this is a breaking change, and we should also probably lock the commit dafny.msbuild depends on.

Reply:
Created https://github.com/dafny-lang/dafny.msbuild/pull/10

Not sure how to test this locally though (or how to obtain the nuget key for publishing)